### PR TITLE
CFUX-201 - Updated Toolbar for Blueprint Editor

### PR DESCRIPTION
**Pull request by dlabrecq**
This PR relocates action buttons in the Blueprint Editor toolbar. It also replaces the "Copy" and "Delete" buttons with Font Awesome icons.

- Replaced the "Copy" and "Delete" buttons with Font Awesome icons.
- Changed "Add Item" button to toggle toolbox open and close.
- Aligned "Blueprint Details" button and "Hide Connections" buttons with masthead.
- Aligned "Filter by name" input with toolbox padding.
- Removed close icon from toolbox per review comments.
- Changed enabled "Copy" and "Delete" icon color per review comments.
- Changed "Blueprint Details" button color per review comments.

JIRA Story: https://patternfly.atlassian.net/browse/CFUX-201

enabled icons

### CODE REVIEW

**Comment by dlabrecq** [comment]
@serenamarie125

**Comment by chriskacerguis** [comment]
@dlabrecq can you please address that Travis failure?

File: client/app/components/blueprints/blueprint-editor/blueprint-editor.html [hunk 1]
```diff
@@ -7,32 +7,39 @@
...
           </button>
           <span class="more-actions">
-              <button class="btn blueprint-details-btn" type="button" ng-click="vm.editDetails()" data-toggle="tooltip"
-                      ng-class="{'disabled': vm.inConnectingMode, 'wait-cursor': vm.loading}"
-                      data-placement="bottom" title="{{'Edit Blueprint Details' | translate}}" translate>Blueprint Details
-              </button>
+              <a id="duplicateItem" ng-click="vm.duplicateSelectedItem()">
+                <span class="pficon fa fa-copy" ng-class="{'disabled': !vm.onlyOneTtemSelected() || vm.inConnectingMode}"
+                      tooltip-append-to-body="true" tooltip-placement="bottom"
+                      tooltip="{{'Duplicate Item (Ctrl-D)'|translate}}" translate>
```
Comment (chriskacerguis) [inline client/app/components/blueprints/blueprint-editor/blueprint-editor.html]: What is the "translate" for?  (the one on the span tag, not the tooltip)

File: client/app/components/blueprints/blueprint-editor/blueprint-editor.html [hunk 2]
```diff
@@ -7,32 +7,39 @@
...
+                </span>
+              </a>
+              <a id="removeItems" ng-click="vm.removeSelectedItemsFromCanvas()">
+                <span class="pficon pficon-delete" ng-class="{'disabled': !vm.itemsSelected() || vm.inConnectingMode}"
+                      tooltip-append-to-body="true" tooltip-placement="bottom"
+                      tooltip="{{'Remove Items From Canvas'|translate}}" translate>
```
Comment (chriskacerguis) [inline client/app/components/blueprints/blueprint-editor/blueprint-editor.html]: What is the "translate" for? (the one on the span tag, not the tooltip)

File: client/app/components/blueprints/blueprint-editor/blueprint-editor.html [hunk 3]
```diff
@@ -100,8 +107,10 @@
                   <a ng-click="vm.searchText = ''"><span class="pficon pficon-close clear-search-text"></span></a>
               </div>
               <div class="close-toolbox">
+                  <!--
```
Comment (chriskacerguis) [inline client/app/components/blueprints/blueprint-editor/blueprint-editor.html]: Why comment this out, as opposed to removing it?

**Comment by dtaylor113** [comment]
Hi Dan,
cloned this branch, tested blueprint editor main buttons and toolbox functionality -all looks good.  Nice work on the new designs!  Unfortunately, these changes broke a unit test:

PhantomJS 2.1.1 (Linux 0.0.0) app.components.blueprints.blueprint-editor.blueprint-editor-directive Blueprints Editor: New Blueprint should handle single and multiple canvas selections FAILED

Thanks,
- Dave

File: client/app/components/blueprints/blueprint-editor/blueprint-editor.html (hunk 2 above)
Comment (dlabrecq) [inline client/app/components/blueprints/blueprint-editor/blueprint-editor.html]: This was left over from the previous button. I'll remove it.

File: client/app/components/blueprints/blueprint-editor/blueprint-editor.html (hunk 3 above)
Comment (dlabrecq) [inline client/app/components/blueprints/blueprint-editor/blueprint-editor.html]: Wasn't certain this wouldn't be needed in the future. I will remove it.

**Comment by dlabrecq** [comment]
Thanks, Dave. Didn't realize the unit tests are run separately.

Fixed the IDs for the unit test and addressed Chris' comments.

**Comment by dtaylor113** [comment]
Thanks for the updates, LGTM :+1:

**Pull request merged**